lib/src/const/language_color.dart

```diff
@@ -1,18 +1,20 @@
 // GENERATED CODE - DO NOT MODIFY BY HAND
-// VERSION OF 2020-02-08T14:23:59.127442
+// VERSION OF 2020-09-22T00:31:27.140510
 
-const languagesColor = <String, String>{
+const kGitHubLanguageColors = <String, String>{
```

Comment:
Why the rename here? This name feels a bit Java. Would you be ok with just `languageColors` ?
```suggestion
const languageColors = <String, String>{
```

Reply:
Hello Rob thanks a lot for the review! 

I used this name to comply with the Flutter naming conventions. (See [here](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k)). 

[Here ](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names)is the more general naming convention for Dart.

I also prefixed it with Github to make it clearer.

So what do you prefer `languagesColor` or `gitHubLanguageColors` or `githubLanguageColors` ?

Reply:
I'm more familiar with the Dart style guide and hadn't seen the Flutter style guide that recommends the leading k. In the Dart style guide it recommends not including prefix letters https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters and I'd prefer that approach since it keeps naming consistent with the rest of this library.
I prefer `languageColors` since I feel that adding `github` is redundant in a library named github, especially if you use a named import like

```dart
import 'package:github/github.dart' as github;
// ...
print(github.languageColors['JavaScript']);

```

Reply:
Also note the change of location of the `s` from
language**s**Color to
languageColor**s**

Reply:
Done.